find_store/__init__.py

```diff
@@ -1,2 +1,2 @@
-from .__main__ import *
+from .entry import *
```

Comment:
renamed to `entry.py` since `__main__` refers to the name of the scope in which top-level code executes (https://docs.python.org/3/library/__main__.html)